internal/endtoend/testdata/mix_param_types/mysql/go/test.sql.go

```diff
@@ -0,0 +1,57 @@
+// Code generated by sqlc. DO NOT EDIT.
+// source: test.sql
+
+package querytest
+
+import (
+	"context"
+)
+
+const countOne = `-- name: CountOne :one
+SELECT count(1) FROM bar WHERE id = ? AND name <> ?
+`
+
+type CountOneParams struct {
+	Name string
+	ID   int64
+}
+
+func (q *Queries) CountOne(ctx context.Context, arg CountOneParams) (int64, error) {
+	row := q.db.QueryRowContext(ctx, countOne, arg.Name, arg.ID)
```

Comment:
This generated code is incorrect. The order of the parameters in the query should be `arg.ID, arg.Name`.

Reply:
I updated the test and the code to pass the test.